docs/Privacy/Explanation/Privacy-Groups.md

```diff
@@ -10,6 +10,13 @@ The Pantheon nodes maintain the public world state for the blockchain and a priv
 The private states contain data that is not shared in the globally replicated world state. Private transactions read 
 and write to the private world state for the privacy group, and read from the public world state.
 
+Contracts have the following levels of access between public and private states:
```

Comment:
It looks like PAN-2831 is not about privacy so this change me be a mistake in this PR.